book/chapters/chapter10/advanced_technical_aspects_of_mlr3.qmd

```diff
@@ -233,10 +232,10 @@ design = benchmark_grid(tsks(c("sonar", "penguins")),
   lrns(c("classif.featureless", "classif.rpart")), rsmp_cv3)
 
 # enable parallelization
-plan("multisession")
+future::plan("multisession")
 
 # run benchmark in parallel
-benchmark(design)$aggregate()[, .(task_id, learner_id, classif.ce)]
+bmr = benchmark(design)
```

Comment:
@mb706 @berndbischl why is output suppressed here?

Reply:
same comment as below for parallel preds